components/scream/src/physics/p3/scream_p3_interface.F90

```diff
@@ -70,14 +71,15 @@ subroutine p3_standalone_init_f90 (q,T,zi,pmid,pdel,ast,naai,npccn) bind(c)
     real(kind=c_real), intent(inout) :: naai(pcols,pver)     ! ice nucleation number
     real(kind=c_real), intent(inout) :: npccn(pcols,pver)    ! liquid activation number tendency
 
-    character(len=100) :: case_title
+    character(len=100) :: case_title, tmp_c1, tmp_c2
 
     integer(kind=c_int) :: i, k
     logical(kind=c_bool) :: masterproc
 
     ! READ inputs from SCM for p3-stand-alone:
     q(:,:,:) = 0.0_rtype
     open(unit=981,file='./data/p3_universal_constants.inp',status='old',action='read')
+    !open(unit=981,file='../build/cmake-bld/scream/tests/scream_p3/data/p3_universal_constants.inp',status='old',action='read')
```

Comment:
If possible, let's not add commented code.

Reply:
For sure, I'll delete.